### ISSUE TITLE
Using pip to build wheels, a.k.a. pep517 bootstrap

### ISSUE THREAD

**Issue by botant** [ionelmc/tox-wheel#4]
Hello, @ionelmc any plans to move forward with support of pep517 / pip?
I'd be willing to help with that, if we know the direction...
Thanks!

**Comment by ionelmc** [comment]
If you're willing to make a PR with that I'd be glad, I didn't have time to do it.

**Comment by botant** [comment]
Cool, I'll have a go in the next few days.

**Comment by botant** [comment]
Hi Ionel,

Apologies for not coming back sooner. Last few months have been crazy.

May I ask a question about the ``patch`` decorator?

```python
@contextmanager
def patch(obj, attr, value):
    original = getattr(obj, attr)
    setattr(obj, attr, value)
    try:
        yield
    finally:
        getattr(obj, attr, original)
```

The call to ``getattr`` inside ``finally``, is that supposed to ``setattr``?

**Comment by ionelmc** [comment]
Oh wow how did I miss that. Yep, it should be setattr.

**Comment by botant** [comment]
I can add the fix to the PR created today, if that is okay with you.

**Comment by ionelmc** [comment]
Please do.

**Comment by ionelmc** [comment]
Just released `0.5.0`.

**Comment by daa** [comment]
Great! Thank you very much.
